### PR TITLE
Add purple Heroku elePHPant

### LINKED ISSUE #73: Add Heroku elePHPant
https://twitter.com/dzuelke/status/916358521851142145

Image of Heroku elePHPant by  David Zuelke‏ (@dzuelke) on Twitter

### PR DESCRIPTION
**Pull request by cookieguru**
Resolves #73

### CODE REVIEW

File: source/_elephpants/2017-10-06-purple-heroku.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    - heroku
+tags:
+    - purple
+sponsor: Heroku
+reverse: Heroku logo
+photos:
+    -
+        file: purple-heroku-0-400x300.jpg
+        link: https://www.flickr.com/photos/atomictaco/38171946464/
+        credit: Rob Allen
```
Comment (asgrim) [inline source/_elephpants/2017-10-06-purple-heroku.md]: Is this Rob Allen's photo?

File: source/_elephpants/2017-10-06-purple-heroku.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    -
+        file: purple-heroku-1-400x300.jpg
+        link: https://www.flickr.com/photos/atomictaco/25016033688/
+        credit: Philip Sharp
```
Comment (asgrim) [inline source/_elephpants/2017-10-06-purple-heroku.md]: Is this Philip Sharp's photo?

File: source/_elephpants/2017-10-06-purple-heroku.md (hunk 1 above)
Comment (cookieguru) [reply source/_elephpants/2017-10-06-purple-heroku.md]: Copy paste fail; thanks for catching

**Comment by cookieguru** [comment]
Fixed edits from @asgrim

**Comment by asgrim** [comment]
Still looks the same..? :)

File: source/_elephpants/2017-10-06-purple-heroku.md (hunk 1 above)
Comment (asgrim) [reply source/_elephpants/2017-10-06-purple-heroku.md]: Also, do we need/have permission from "atomictaco" that these images can be used?

**Comment by cookieguru** [comment]
Missed a step, now I'm sure it's there :)

File: source/_elephpants/2017-10-06-purple-heroku.md (hunk 1 above)
Comment (cookieguru) [reply source/_elephpants/2017-10-06-purple-heroku.md]: I have several personas online; I am also atomictaco.
Regardless, the photos are CC-BY-SA and are only linked as per request from [CONTRIBUTING](https://github.com/philipsharp/afieldguidetoelephpants/blob/master/CONTRIBUTING.md).

Comment (asgrim) [reply source/_elephpants/2017-10-06-purple-heroku.md]: :+1: awesome! :)

**Comment by philipsharp** [comment]
You'll need to add `heroku` as a key to the subspecies list in `app/config/sculpin_site.yml`. The common name should be "Heroku" but you'll have to chose a Latin name. There was some discussion here last night: https://twitter.com/dzuelke/status/938529594772197376

**Comment by philipsharp** [comment]
Closed by mistake

**Pull request merged**